src/ypspur_ros.cpp

```diff
@@ -1005,10 +1014,11 @@ class YpspurRosNode
         }
         pubs_["joint"].publish(joint);
 
+        tf2::Quaternion quat_tf;
         for (unsigned int i = 0; i < joints_.size(); i++)
         {
-          joint_trans[i].transform.rotation =
-              tf::createQuaternionMsgFromYaw(joint.position[i]);
+          quat_tf.setRPY(0, 0, joint.position[i]);
+          joint_trans[i].transform.rotation = tf2::toMsg(quat_tf);
```

Comment:
or
```suggestion
          joint_trans[i].transform.rotation = tf2::toMsg(tf2::Quaternion(0, 0, joint.position[i]));
```
would be more simple.